control_panel_api/permissions.py

```diff
@@ -28,5 +31,23 @@ class S3BucketPermissions(IsSuperuser):
     pass
 
 
-class UserPermissions(IsSuperuser):
+class IsSelf(BasePermission):
+
+    def has_permission(self, request, view):
+        if super().has_permission(request, view):
+            return True
+
+        if hasattr(request.user, 'auth0_id'):
```

Comment:
Nice, as far I can tell this should also handle the case where the user is not even logged in so `request.user` is `None`, or something like that...am I wrong?

Reply:
Yes. When the user is not logged in, `request.user` is an instance of `AnonymousUser` which does not have an `auth0_id` attribute.